configure.ac

```diff
@@ -183,6 +183,9 @@ AM_CONDITIONAL(HAVE_STRCHRNUL, [test "x${ac_cv_func_strchrnul}" = xyes])
 AC_CHECK_FUNCS([strnlen])
 AM_CONDITIONAL(HAVE_STRNLEN, [test "x${ac_cv_func_strnlen}" = xyes])
 
+AC_CHECK_FUNCS([prlimit])
+AM_CONDITIONAL(HAVE_PRLIMIT, [test "x${ac_cv_func_prlmit}" = xyes])
```

Comment:
this AM_CONDITIONAL is pointless as it isn't used in the makefiles.  just delete it.